include/filters/filter_base.hpp

```diff
@@ -122,6 +122,7 @@ class FilterBase
   virtual rcl_interfaces::msg::SetParametersResult reconfigureCB(
     std::vector<rclcpp::Parameter> parameters)
   {
+    (void)parameters;
```

Comment:
Ah thanks for fixing this!  Could you add a comment explaining the new line? Just something short like "This avoids unused parameter warning"

Reply:
sure, done!